graphql-kotlin-schema-generator/build.gradle.kts

```diff
@@ -5,7 +5,7 @@ val graphQLJavaVersion: String by project
 val jacksonVersion: String by project
 val kotlinVersion: String by project
 val kotlinCoroutinesVersion: String by project
-val rxjavaVersion = "3.0.0"
+val rxjavaVersion: String by project
```

Comment:
maybe we should update our tests to create custom Reactor Monad and drop RxJava altogether?

Reply:
*just a side note for future

Reply:
If we have it as an official example in our documentation I like having it included in our test dependencies. 

But as you mentioned we could just edit our docs and examples to just use reactor/spring wrappers and comment that they can work for any custom wrappers you want to include